### PR TITLE
[Merged by Bors] - fix: incorrectly formatted MEV link in Lighthouse Book

### PR DESCRIPTION
**Pull request by BrendanTimmons**
## Issue Addressed

N/A

## Proposed Changes

Simply fix the incorrect formatting on markdown link.

### CODE REVIEW

**Review by realbigsean — APPROVED** [review]
Thanks a lot!

**Comment by michaelsproul** [comment]
@BrendanTimmons Could you please click through the contributor license agreement that the bot posted? :pray: https://github.com/sigp/lighthouse/pull/3434#issuecomment-1207607740. CI won't allow us to merge without that

Thanks!

**Comment by BrendanTimmons** [comment]
> @BrendanTimmons Could you please click through the contributor license agreement that the bot posted? 🙏 [#3434 (comment)](https://github.com/sigp/lighthouse/pull/3434#issuecomment-1207607740). CI won't allow us to merge without that
> 
> Thanks!

Done!

File: book/src/builders.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ knowledge of the transactions included in the block. This enables Lighthouse to
 transaction gathering/ordering within a block to parties specialized in this particular task. For
 economic reasons, these parties will refuse to reveal the list of transactions to the validator
 before the validator has committed to (i.e. signed) the block. A primer on MEV can be found
-[here]([MEV](https://ethereum.org/en/developers/docs/mev/)).
+[here](https://ethereum.org/en/developers/docs/mev/).
```
Comment (michaelsproul) [inline book/src/builders.md]:
```suggestion
[here](https://ethereum.org/en/developers/docs/mev).
```

**Pull request closed without merging**